packages/frontend/src/features/CohortBuilder/QueryExpressionSection.tsx

```diff
@@ -275,9 +290,7 @@ const QueryExpressionSection : React.FC<QueryExpressionSectionProps> = ({
               <p
                 data-testid="text-no-active-cohort-filter"
                 className="font-content"
-              >
-                No filters currently applied.
-              </p>
+              ></p>
```

Comment:
Since there's no text on it now, can we add a "Filters" title on this component to clarify to users?